kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/utils/FieldNameSanitizerTest.java

```diff
@@ -75,4 +77,10 @@ public void testInvalidSymbol() {
     // Validate map size.
     assertEquals(5, sanitizedMap.size());
   }
+
+  @Test (expected = ConnectException.class)
+  public void testNullKey() {
+    testMap.put(null, "val");
```

Comment:
What if you call `testMap.put("abc", null)`?

Reply:
This also passes.

Reply:
I added a test for this, and a null check for value.